docker/runtime/nodejs/event-trigger/kubeless.js

```diff
@@ -47,4 +48,4 @@ kafkaConsumer.on('message', (message) => {
   }
 });
 
-app.listen(8080);
+app.listen(funcPort);
```

Comment:
note that `process.env.FUNC_PORT` will be a string while `8080` is a int. Does app.listen works if it is a string?

Reply:
That is an assumption I made, and I have not tested it yet.

Reply:
I also could not get the integration tests to run on a Mac, so that slows things down for me too as I have to run those tests manually.

Reply:
I tried it in a node console. It works:
```
> const express = require('express');
undefined
> app = express();
> app.listen('8080');
Server {
  domain:
   Domain {
...
```

Reply:
BTW, I opened an issue to document/make it easier to test things in OS X

Reply:
Thanks,  was planning to do that.  I will put more details in that issue.

Reply:
fixed by https://github.com/kubeless/kubeless/pull/424/commits/67a9f67b176a05ca806ace14292b630186582551